pkg/test/client.go

```diff
@@ -182,6 +192,22 @@ func toMap(obj runtime.Object) (map[string]interface{}, error) {
 	return m, nil
 }
 
+func getAllToolchainResources(s *runtime.Scheme) []client.Object {
+	toolchainObjs := make([]client.Object, 0)
+	KindToTypeMap := s.KnownTypes(toolchainv1alpha1.GroupVersion)
+	var kinds []string
+	for key := range KindToTypeMap {
```

Comment:
any reason for starting with the capital letter?
```suggestion
	kindToTypeMap := s.KnownTypes(toolchainv1alpha1.GroupVersion)
	var kinds []string
	for key := range kindToTypeMap {
```